src/attached_probe.cpp

```diff
@@ -379,7 +379,10 @@ static int sym_address_cb(const char *symname,
 {
   struct symbol *sym = static_cast<struct symbol *>(p);
 
-  if (sym->address >= start && sym->address < (start + size)) {
+  // When size is 0, then [start, start + size) = [start, start) = ø.
```

Comment:
This can happen when the symbol has no size or an unknown size.
I'm not sure a 0-length symbol is created, but our `tests/runtime/probe - uprobe_zero_size` uses the `_init` symbol to validate bpftrace's behaviour. By default, it won't let you probe such symbol, but you can pass `--unsafe` to force it.